app/assets/javascripts/discourse/templates/preferences/notifications.hbs

```diff
@@ -39,14 +39,16 @@
   </div>
 {{/if}}
 
-<div class="control-group muting">
-  <label class="control-label">{{i18n 'user.users'}}</label>
-  <div class="controls category-controls">
-    <label>{{d-icon "d-muted" class="muted icon"}} {{i18n 'user.muted_users'}}</label>
-    {{user-selector excludeCurrentUser=true usernames=model.muted_usernames class="user-selector"}}
+{{#unless siteSettings.ignore_user_enabled}}
+  <div class="control-group muting">
+    <label class="control-label">{{i18n 'user.users'}}</label>
+    <div class="controls category-controls">
+      <label>{{d-icon "d-muted" class="muted icon"}} {{i18n 'user.muted_users'}}</label>
+      {{user-selector excludeCurrentUser=true usernames=model.muted_usernames class="user-selector"}}
+    </div>
+    <div class="instructions">{{i18n 'user.muted_users_instructions'}}</div>
```

Comment:
where is the `</div>` ?

I think you can improve html formatting.

Reply:
Which `</div>`?

I can see the it at [the end of the line](https://github.com/discourse/discourse/pull/7107/files#diff-203c4b2a392348e0564b713df30f9d35R49).

`....{{i18n 'user.muted_users_instructions'}}   </div>`

Also, this is really unchanged since I just wrapped into a `{{#unless siteSettings.ignore_user_enabled}}`